### ISSUE TITLE
Installing node-inspector

### ISSUE THREAD

**Issue by karaxuna** [jxcore/jxcore#725]
Installing node-inspector:

```bash
jx install node-inspector -g
```

Results error:

```bash
LINK : fatal error LNK1104: cannot open file 'libcpmtd.lib' [C:\Users\...\
AppData\Roaming\npm\node_modules\node-inspector\node_modules\ws\node_modules\bu
fferutil\build\bufferutil.vcxproj]
gyp ERR! build error
gyp ERR! stack Error: `C:\Program Files (x86)\MSBuild\14.0\bin\msbuild.exe` fail
ed with exit code: 1
gyp ERR! stack     at ChildProcess.onExit (C:\Users\...\.jx\npm\node_module
s\node-gyp\lib\build.js:271:23)
gyp ERR! stack     at ChildProcess.emit (events.js:85:17)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (child_process.js:826:
12)
gyp ERR! System Windows_NT 6.3.9600
gyp ERR! command "jx" "C:\\Users\\...\\.jx\\npm\\node_modules\\node-gyp\\bi
n\\node-gyp.js" "rebuild"
gyp ERR! cwd C:\Users\...\AppData\Roaming\npm\node_modules\node-inspector\n
ode_modules\ws\node_modules\bufferutil
gyp ERR! node -v v0.10.40
gyp ERR! node-gyp -v v0.13.1
gyp ERR! not ok
```

**Comment by ktrzeciaknubisa** [comment]
Which jx version did you use? I don't reproduce this on Win 7 x64 with jx 0.3.1.0.

I had another problem though with 0.3.0.7, but this is already fixed in 0.3.1.0 with https://github.com/jxcore/jxcore/commit/2393bed306a4e9dd952248eda012dbd3e1558c91 :

```bash
'C:\Program' is not recognized as an internal or external command,
```

**Comment by karaxuna** [comment]
I'm using `0.3.1.0`, built with `vcbuild.bat debug`. Took `jx.exe` and replaced old `jx.exe` in program files.

**Comment by obastemur** [comment]
probably you were using either `--engine-chakra` or `--engine-mozilla` ?

**Comment by karaxuna** [comment]
```bash
$ jx -jsv
Google V8 v3.14.5.9
```

**Comment by obastemur** [comment]
Guess the reason is that we need to update the header file package jx307 with latest header files etc. and server under jx310. @ktrzeciaknubisa ? AFAIK, we should also update internal references for file

**Comment by karaxuna** [comment]
Same happens with `vcbuild.bat x86` without debug flag, just checked. @obastemur is it reproducable for you?

**Comment by obastemur** [comment]
no need to push it more, You got the header files for 307 under node-gyp.. That's the reason. @ktrzeciaknubisa gonna update the header files.

**Comment by ktrzeciaknubisa** [comment]
@obastemur I'm not sure why this would matter here? When I tested this, I even removed `~/.jx` and `~/.node-gyp` to make sure it gets downloaded, but still the installation goes fine. @karaxuna You can try that too.

@karaxuna Is your system x86?

**Comment by karaxuna** [comment]
@ktrzeciaknubisa my system is x64

**Comment by ktrzeciaknubisa** [comment]
Ok, on my VS 2015 (Windows 10) I have a problem too, however slightly different:

> C:\Program Files (x86)\MSBuild\Microsoft.Cpp\v4.0\V110\Microsoft.Cpp.Platform.targets(44,5): error MSB8020: The builds tools for v120 (Platform Tool
set = 'v120') cannot be found. To build using the v120 build tools, either click the Project menu or right-click the solution, and then select "Upda
te VC++ Projects...". Install v120 to build using the v120 build tools. [C:\Users\Krzysztof\AppData\Roaming\npm\node_modules\bufferutil\build\buffer
util.vcxproj]


And that also happens with node's npm: 

```bash
npm install node-inspector -g
```

Can node-gyp replace the MS tools version?

**Comment by obastemur** [comment]
We might copy msvs.py etc. files from our tools folder to deps/npm/node-gyp. I had updated them for VS2015

**Comment by obastemur** [comment]
@karaxuna were you able to build this with node / npm ? (force rebuild)

**Comment by karaxuna** [comment]
@obastemur  yes, i installed with `npm install node-inspector -g` successfully

**Comment by obastemur** [comment]
@karaxuna  That doesn't rebuild the binary..
